src/gui/properties/speedplotview.cpp

```diff
@@ -231,7 +293,7 @@ void SpeedPlotView::paintEvent(QPaintEvent *)
     // draw graphs
     rect.adjust(3, 0, 0, 0); // Need, else graphs cross left gridline
 
-    double yMultiplier = (maxY == 0) ? 0.0 : static_cast<double>(rect.height()) / maxY;
+    double yMultiplier = (niceScale == 0.0) ? 0.0 : static_cast<double>(rect.height()) / niceScale;
```

Comment:
You don't need cast to double since niceScale is double.

Reply:
Fixed.